src/game/game-object.ts

```diff
@@ -10,6 +10,8 @@ export abstract class GameObject {
 
     abstract update(dt: number, context: GameContext): void;
 
+    abstract getForward(): Vector;
```

Comment:
I didn't notice that base game-object doesn't have the `forward` property. In that case the abstract method does't make sense 😇 , but when we're at it, why not add it and provide a default implementation of the `getForward` method?